src/main/java/com/nike/cerberus/domain/cloudformation/EnvironmentConfigBucketParameters.java

```diff
@@ -19,17 +19,27 @@
 /**
  * Represents the base stack inputs.
  */
-public class BaseParameters {
+public class EnvironmentConfigBucketParameters {
```

Comment:
- [ ] rename to RegionSpecificBase ?
Includes S3 Bucket for config and KMS CMK for config and CMS secure Data.